spec/index.bs

```diff
@@ -3093,13 +3094,23 @@ enum GPUPrimitiveTopology {
 dictionary GPURasterizationStateDescriptor {
     GPUFrontFace frontFace = "ccw";
     GPUCullMode cullMode = "none";
+    // Enable depth clamping (requires "depth-clamp" extension)
+    boolean clampDepth = false;
```

Comment:
Should we have the depthClampBias gated under the same feature or are the availibitiy of the two mechanisms different enough that we want them separate?

Reply:
Technically the mechanisms are different. Looking at VulkanDB, the support seems close, except on Android: https://vulkan.gpuinfo.org/listfeatures.php?platform=android . On Metal, the depth bias clamping is non-optional, but  "Depth clipping is supported on all macOS GPU families and iOS family 4 and later." (I think they meant "clamping" instead of "clipping"?)

Therefore, it seems fine to me to have `depthBiasClamp` as required to be supported in the core, doesn't need an extension.

Reply:
Yeah it seems ok to have depthBiasClamp in core.